niceman/distributions/debian.py

```diff
@@ -89,26 +89,26 @@ class APTSource(SpecObject):
 _register_with_representer(APTSource)
 
 
-@attr.s(slots=True)
+@attr.s(slots=True, frozen=True, cmp=False, hash=True)
 class DEBPackage(Package):
     """Debian package information"""
-    name = attr.ib()
+    name = attr.ib(hash=True)
     # Optional
-    source_name = attr.ib(default=None)
+    source_name = attr.ib(default=None, hash=False)
     upstream_name = attr.ib(default=None)
 
-    version = attr.ib(default=None)
-    architecture = attr.ib(default=None)
-    size = attr.ib(default=None)
-    md5 = attr.ib(default=None)
-    sha1 = attr.ib(default=None)
-    sha256 = attr.ib(default=None)
-    source_version = attr.ib(default=None)
-    versions = attr.ib(default=None)
-    install_date = attr.ib(default=None)
+    version = attr.ib(default=None, hash=True)
```

Comment:
it is hard(er) to differentiate between those with True or False.  The default for `hash` seems to be True, so I would prefer if `, hash=True` ones were removed, then it would be easier to see which fields we consider not important